frontend/src/components/userMenu.vue

```diff
@@ -53,6 +53,13 @@ export default {
   cursor: pointer;
 }
 
+.animation {
+  transition: $transition;
+  &:hover {
+    transform: $transform
+  }
+}
+
```

Comment:
Nice! Je kan deze combinatie ook in theme zetten voor een small en big transform omdat je hem telkens redefined. DRY Principe